docs/migrating/migrating-1.x-to-2.x.md

```diff
@@ -64,24 +65,24 @@ If your bundle relied on the `accessToken` and `refreshToken` properties availab
 import type NodeCG from '@nodecg/types';
 
 export = (nodecg: NodeCG.ServerAPI) => {
-	nodecg.on('login', (user) => {
-		console.log(user.id, user.name);
-	});
-	
-	nodecg.on('logout', (user) => {
-		console.log(user.id, user.name);
-	});
-
-	nodecg.mount((req, _res, _next) => {
-		if (req.user) {
-			const ident = req.user.identities[0];
-			switch (ident.provider_type) {
-				case 'discord':
-				case 'twitch':
-					console.log(ident.provider_access_token, ident.provider_refresh_token);
-			}
-		}
-	});
+ nodecg.on('login', (user) => {
+  console.log(user.id, user.name);
+ });
+ 
+ nodecg.on('logout', (user) => {
+  console.log(user.id, user.name);
+ });
+
+ nodecg.mount((req, _res, _next) => {
+  if (req.user) {
+   const ident = req.user.identities[0];
+   switch (ident.provider_type) {
+    case 'discord':
+    case 'twitch':
+     console.log(ident.provider_access_token, ident.provider_refresh_token);
```

Comment:
It's because the markdown linter is setup to use spaces. I use tabs all the time but for markdown it gets weird because GitHub makes tabs super long so reading a markdown file here when something is using tabs is bad. I think we should probably use spaces. It also seems that docusaurus converts it to spaces as well if you check it out on the website.

Tabs (Current):
![image](https://github.com/nodecg/docs/assets/19513845/afebd514-dd66-4a72-966f-9125c67de162)

Spaces:
![image](https://github.com/nodecg/docs/assets/19513845/08479735-a2ff-4ad9-83a5-f564808c3668)

Reply:
It doesn't seem to be working on GitHub's new site navigation stuff because I have mine set to 4 and it's still super wide. Changing it to 2 did nothing as well.

Reply:
Yeah not a fan of single space indenting either. Much prefer 4 space indents, either Simon can check if the markdownlinter will allow/can be configured with multiple space tabs or I can check it when I get home tonight.

Reply:
Not a fan of single space indenting either. So I actually already configured the linter to use 3 spaces (as that was what I found on some code-blocks already and is the minimum to not confuse the linter and break the MD029/ol-prefix rule).
But I also usually use 4, will change that.
So don't actually know why here it's only one.
Working on it...